packages/react-components/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -36,82 +49,132 @@ const useStyles = makeStyles({
   filledInteractive: {
     cursor: 'pointer',
     backgroundColor: tokens.colorNeutralBackground1,
-    ...shorthands.borderColor(tokens.colorTransparentStroke),
     boxShadow: tokens.shadow4,
 
+    '::after': {
+      ...shorthands.borderColor(tokens.colorTransparentStroke),
```

Comment:
In retrospect, I can add a style that has this:
```
//pending style name™️
interactiveNoOutline: {
  ':hover::after': {
    ...shorthands.borderColor(tokens.colorTransparentStrokeInteractive),
  },
  ':active::after': {
    ...shorthands.borderColor(tokens.colorTransparentStrokeInteractive),
  },
},
```
This would be applied to all interactive appearances, except for `outline` as this uses different colors.

If you mean that just to reduce duplications of `...shorthands.borderColor(tokens.colorTransparentStrokeInteractive)`, I guess we could extract it into a variable, but the styles would also become less readable IMO.

To clarify, my fear is the detachment of what's happening becoming so great that the maintenance of this code becomes a big burden. Having pseudo elements for borders already puts such a complexity to our styles that I feel "optimizing" would create even more entropy when understanding what's going on.

WDYT about these approaches? Let me know if you have other suggestions too

Reply:
> In retrospect, I can add a style that has this:

that would be definitely more preferable then the current approach IMO

> If you mean that just to reduce duplications of ...shorthands.borderColor(tokens.colorTransparentStrokeInteractive), I guess we could extract it into a variable, but the styles would also become less readable IMO.

No I meant if there is some API to create custom mixins (like one can do in [sass](https://sass-lang.com/documentation/at-rules/mixin) or [vanilla-extract keyframes](https://vanilla-extract.style/documentation/styling-api/#keyframes) etc)

> Having pseudo elements for borders already puts such a complexity to our styles that I feel "optimizing" would create even more entropy when understanding what's going on.

👍

Reply:
> that would be definitely more preferable than the current approach IMO

I'll apply the changes then, let's see how it turns out in practice.

> No I meant if there is some API to create custom mixins (like one can do in [sass](https://sass-lang.com/documentation/at-rules/mixin) or [vanilla-extract keyframes](https://vanilla-extract.style/documentation/styling-api/#keyframes) etc)

Not that I'm aware of unfortunately. 😞

Reply:
I'm personally not a fan of how it looks. I think I'll need to do some refactoring of the styles for it to look more readable but that's out of this scope.
Let me know if you think I should leave it as is or revert the last commit @Hotell.

Reply:
lets move forward with lats change. ty